.github/reports/workflow-testing/2026-08-04-script-organization-concern.md

```
@@ -0,0 +1,187 @@
+**Risk:** Limits reusability if scripts later need to be shared
+
+---
+
+### Option B: Move Reusable Scripts to Root (Recommended)
+
+**Decision:** Create `scripts/` folder for portable scripts
+
+**Actions Needed:**
+1. ✓ Audit scripts in `.github/scripts/` → classify as portable vs. control-plane-specific
+2. ✓ Create `scripts/` folder structure:
+   ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)